tests/api.c

```diff
@@ -43264,6 +43264,142 @@ static int test_wolfSSL_X509_NAME_ENTRY(void)
     return res;
 }
 
+/* Note the lack of wolfSSL_ prefix...this is a compatability layer test. */
+static int test_GENERAL_NAME_set0_othername(void) {
+    int res = TEST_SKIPPED;
+#if defined(OPENSSL_EXTRA) && !defined(NO_CERTS) && \
+    defined(WOLFSSL_CERT_GEN) && defined(WOLFSSL_CERT_REQ) && \
+    defined(WOLFSSL_CUSTOM_OID) && defined(WOLFSSL_ALT_NAMES) && \
```

Comment:
Am not seeing WOLFSSL_CUSTOM_OID in configure.ac or in the testing flags for CI. Did I miss where the macro is being defined somewhere and this test being ran with CI testing.

Reply:
maybe not in this PR but please either a) add WOLFSSL_CUSTOM_OID to --enable-all or b) add a specific WOLFSSL_CUSTOM_OID test to our CI enable flag list